examples/quad_android/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
```

Comment:
another option, not necessarily a better one, is to have `examples/src/lib.rs`, so that the project is a `cdylib` library with binaries. The library would only be doing stuff on Android at this moment, but maybe we can find more use for this later?

Reply:
`cargo apk` allows run apk file only from cargo workspace root. But cargo doesn't allow pass features in cargo workspace root.

```bash
error: --features is not allowed in the root of a virtual workspace
```

Reply:
This is error already fixed in `cargo-apk`. I will try to remove more code duplication with `examples/src/lib.rs`.